### PR TITLE
Updated cs-CZ translation

### PR DESCRIPTION
**Pull request by JanRajnoha**

### CODE REVIEW

File: src/Notepads/Strings/cs-CZ/Manifest.resw [hunk 1]
```diff
@@ -118,199 +118,200 @@
...
   </data>
   <data name="ZshellConfigFileDisplayName" xml:space="preserve">
-    <value>Z shell Config File</value>
+    <value>Konfigurační soubor Z shell</value>
     <comment>zshrc</comment>
   </data>
   <data name="ZshellHistoryFileDisplayName" xml:space="preserve">
-    <value>Z shell History File</value>
+    <value>Soubor historie Z shell</value>
+
```
Comment (JasonStein) [inline src/Notepads/Strings/cs-CZ/Manifest.resw:314]: Can you help remove this empty space?

Comment (JanRajnoha) [reply src/Notepads/Strings/cs-CZ/Manifest.resw]: Done

Comment (JasonStein) [reply src/Notepads/Strings/cs-CZ/Manifest.resw]: @JanRajnoha  I just added few more string tonight. Can you sync your branch with master to make below changes?:
image

Comment (JanRajnoha) [reply src/Notepads/Strings/cs-CZ/Manifest.resw]: Done

**Pull request merged**